resources/nl-core/nl-core-Payer.xml

```
@@ -0,0 +1,60 @@
+  <purpose value="A derived profile from [zib-Payer](http://nictiz.nl/fhir/StructureDefinition/zib-Payer) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Coverage" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-Payer" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Coverage">
+      <path value="Coverage" />
+      <alias value="nl-core-Payer" />
+    </element>
+    <element id="Coverage.beneficiary">
+      <path value="Coverage.beneficiary" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Patient" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient" />
+      </type>
+    </element>
+    <element id="Coverage.payor:payerPerson">
+      <path value="Coverage.payor" />
+      <sliceName value="payerPerson" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Organization" />
```

I think you might want to remove the base references here because you are in a slice definition. It is still possible to reference a Organization in another slice for example. Moreover, Organization does not fit into the payerPerson definition. The tooling errors can be silenced.

---

Do not agree with this one. The zib definition explicitly says that a PayerPerson can be an organization: 
> [...] In this a person is a natural person or **a juridical person, such as an organization, municipality**, etc.

My intention is that if the reference is not conforming to profile zib-Payer-Organization, then it must fall under the slice payerPerson. That is also why I want to use resolve() instead of type.

---

Sorry! You are right. Didn't read the definition well. 

I don't think the zib-Payer-Organization provides a lot of discriminative value, as nothing is fixed or so. An Organization instance will be valid against the profile as well as the base?

---

And should http://hl7.org/fhir/StructureDefinition/Organization then also not be our nl-core-profile for it?

---

FIXED by spliting the profile into two based on the two containers InsuranceCompany and PayerPerson